deploy_docker/000-default.conf

```diff
@@ -15,6 +15,14 @@
 	# modules, e.g.
 	#LogLevel info ssl:warn
 
+
+	CustomLog ${APACHE_LOG_DIR}/access.log combined "expr=%{REQUEST_STATUS} > 200"
```

Comment:
1. Does this require that `combined` already be defined in a `LogFormat` expression?
2. Does `"expr=%{REQUEST_STATUS} > 200"` filter to just successful requests? If so, do you want to log unsuccessful requests somewhere else, like "error.log"?

Reply:
1. This works with or without defining `combined` in a `LogFormat` expression. Would you prefer I remove the definition?

2. This will only log requests that have a HTTP status code above 200. For the errors, they are logged to `error.log` which is a reference to STDERR. The unsuccessful requests would also show up in the browser